drivers/timer/Kconfig.mcux_lptmr

```diff
@@ -5,22 +5,11 @@
 
 config MCUX_LPTMR_TIMER
 	bool "MCUX LPTMR timer"
-	default y
-	depends on DT_HAS_NXP_KINETIS_LPTMR_ENABLED || \
-		   MCUX_KINETIS_LPTMR
+	default n
```

Comment:
Just curious. Why was its default value changed from y to n?

Reply:
read the commit message, its to avoid build errors because systick is also enabled by default

Reply:
You are right. I have read only the PR message.
More info is in the commit messages.